assets/sass/2_fragments/_bulk-actions.scss

```diff
@@ -0,0 +1,44 @@
+.bulk-action {
+    position: relative;
+    @include clearfix;
+    @include typeGamma;
+    font-weight: 700;
+
+    .bulk-action-primary {
+        padding: $base-spacing $sm-spacing;
+        position: relative;
+
+        @include media($medium) {
+                padding: 24px 0px;
+            }
+    }
+
+    .bulk-action-secondary {
+
+        font-size: .857142857em;
+        padding: $base-spacing $sm-spacing;
+        padding-right: 10px;
+        position: absolute;
+        right: 10px;
+        top: 0px;
+        width: 50%;
+        z-index: $body;
+    
+        @include text-align(right);
+        @include right($sm-spacing);
+
+        @include media($medium) {
+            @include right($base-spacing);
+        }
+
+            @include media($medium) {
+                right: 0px;
+                padding-right: 0px;
+            }
+    }
+
+    .button-link {
+        text-decoration: underline;
+        font-weight: 700;
+    }
+}
```

Comment:
RTL support is needed here but I will need to review all of this at a later date.